management/cmd/management.go

```diff
@@ -184,14 +188,96 @@ func loadTLSConfig(certFile string, certKey string) (*tls.Config, error) {
+
+	if _, err = io.Copy(dstfd, srcfd); err != nil {
+		return err
+	}
+	if srcinfo, err = os.Stat(src); err != nil {
+		return err
+	}
+	return os.Chmod(dst, srcinfo.Mode())
+}
+
+func cpDir(src string, dst string) error {
+	var err error
+	var fds []os.FileInfo
+	var srcinfo os.FileInfo
+
+	if srcinfo, err = os.Stat(src); err != nil {
+		return err
+	}
+
+	if err = os.MkdirAll(dst, srcinfo.Mode()); err != nil {
+		return err
+	}
+
+	if fds, err = ioutil.ReadDir(src); err != nil {
+		return err
+	}
+	for _, fd := range fds {
+		srcfp := path.Join(src, fd.Name())
+		dstfp := path.Join(dst, fd.Name())
+
+		if fd.IsDir() {
+			if err = cpDir(srcfp, dstfp); err != nil {
+				log.Fatalf("Failed to copy from %s to %s; %v", srcfp, dstfp, err)
+			}
+		} else {
+			if err = cpFile(srcfp, dstfp); err != nil {
+				log.Info(err)
+				log.Fatalf("Failed to copy from %s to %s; %v", srcfp, dstfp, err)
+			}
+		}
+	}
+	return nil
+}
+
+func initDefault() {
 	mgmtCmd.Flags().IntVar(&mgmtPort, "port", 33073, "server port to listen on")
-	mgmtCmd.Flags().StringVar(&mgmtDataDir, "datadir", "/var/lib/wiretrustee/", "server data directory location")
-	mgmtCmd.Flags().StringVar(&mgmtConfig, "config", "/etc/wiretrustee/management.json", "Wiretrustee config file location. Config params specified via command line (e.g. datadir) have a precedence over configuration from this file")
+	mgmtCmd.Flags().StringVar(&mgmtDataDir, "datadir", "/var/lib/netbird/", "server data directory location")
+	mgmtCmd.Flags().StringVar(&mgmtConfig, "config", "/etc/netbird/management.json", "Netbird config file location. Config params specified via command line (e.g. datadir) have a precedence over configuration from this file")
 	mgmtCmd.Flags().StringVar(&mgmtLetsencryptDomain, "letsencrypt-domain", "", "a domain to issue Let's Encrypt certificate for. Enables TLS using Let's Encrypt. Will fetch and renew certificate, and run the server with TLS")
 	mgmtCmd.Flags().StringVar(&certFile, "cert-file", "", "Location of your SSL certificate. Can be used when you have an existing certificate and don't want a new certificate be generated automatically. If letsencrypt-domain is specified this property has no effect")
 	mgmtCmd.Flags().StringVar(&certKey, "cert-key", "", "Location of your SSL certificate private key. Can be used when you have an existing certificate and don't want a new certificate be generated automatically. If letsencrypt-domain is specified this property has no effect")
-
 	rootCmd.MarkFlagRequired("config") //nolint
+}
+
+func migrateToNetbird() {
+	if err := cpDir("/var/lib/wiretrustee", "/var/lib/netbird"); err != nil {
+		log.Fatal(err)
+	}
+
+	if err := cpDir("/etc/wiretrustee/", "/etc/netbird/"); err != nil {
+		log.Fatal(err)
+	}
 
+	if err := cpFile("/etc/wiretrustee/management.json", "/etc/netbird/management.json"); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func init() {
+	_, old := os.Stat("/var/lib/wiretrustee")
+	_, new := os.Stat("/var/lib/netbird")
```

Comment:
If a user provides a custom path for any of the paths (logs, configuration, store, and SSL) you shouldn't move them to the default paths.

Reply:
done.